CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [FEATURE] Add buckets per tenant that archive truly resource isolation (`--s3.buckets-prefix` globally) each tenants will create one bucket with prefix. [#2292](https://github.com/grafana/loki/issues/2292)
```

Comment:
Cortex release 1.8.0 is now in progress. Could you please rebase master and move the CHANGELOG entry under the master / unreleased section?

Reply:
ok

Reply:
> Cortex release 1.8.0 is now in progress. Could you please rebase master and move the CHANGELOG entry under the master / unreleased section?

done!

Reply:
done!